morse.rb

```diff
@@ -0,0 +1,34 @@
+@morse_code = {
+  '.-' => 'a',
+  '-...' => 'b',
+  '-.-.' => 'c',
+  '-..' => 'd',
+  '.' => 'e',
+  '..-.' => 'f',
+  '--.' => 'g',
+  '....' => 'h',
+  '..' => 'i',
+  '.---' => 'j',
+  '-.-' => 'k',
+  '.-..' => 'l',
+  '--' => 'm',
+  '-.' => 'n',
+  '---' => 'o',
+  '.--.' => 'p',
+  '--.-' => 'q',
+  '.-.' => 'r',
+  '...' => 's',
+  '-' => 't',
+  '..-' => 'u',
+  '...-' => 'v',
+  '.--' => 'w',
+  '-..-' => 'x',
+  '-.--' => 'y',
+  '--..' => 'z'
+}
+
+def decode_char(_char)
+  puts @morse_code[char].upcase
+end
+
+decode_char('.-')
```

Comment:
- [ ] according to the requirements apart from `decode_char` you should:
- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. `decode_word("-- -.--")` returns "MY").
- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g. ` decode("-- -.--   -. .- -- .")` returns "MY NAME").)